include/SDL3/SDL_gpu.h

```diff
@@ -44,49 +44,174 @@ extern "C" {
 
 /* Type Declarations */
 
+/**
+ * An opaque handle representing the SDL_GPU context.
+ *
+ * \since This struct is available since SDL 3.0.0
+ */
 typedef struct SDL_GPUDevice SDL_GPUDevice;
+
+/**
+ * An opaque handle representing a buffer.
+ * Used for vertices, indices, indirect draw commands, and general compute data.
+ *
+ * \since This struct is available since SDL 3.0.0
+ */
 typedef struct SDL_GPUBuffer SDL_GPUBuffer;
+
+/**
+ * An opaque handle representing a transfer buffer.
+ * Used for transferring data to and from the device.
+ *
+ * \since This struct is available since SDL 3.0.0
```

Comment:
You might want to add `\sa SDL_Function`, for functions that reference this type.